tests/commands/config.py

```diff
@@ -0,0 +1,785 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import json
+
+import pytest
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from pootle.core.delegate import config
+from pootle_project.models import Project
+
+
+def _repr_value(value):
+    if not isinstance(value, (str, unicode)):
+        value_class = type(value).__name__
+        return (
+            "%s(%s)"
+            % (value_class,
+               json.dumps(value)))
+    return value
+
+
+def _test_config_get(out, key, model=None, instance=None, asjson=False):
+    expected = ""
+    conf = config.get(model, instance=instance)
+    try:
+        expected = conf.get_config(key)
+        expected_class = type(expected).__name__
+        expected = json.dumps(expected)
+        if not asjson:
+            expected = "%s(%s)" % (expected_class, expected)
```

Comment:
in string mode it wraps the var with the type, to disambiguate between a string that happens to be valid json, from some actual json